src/utilities.jl

```diff
@@ -59,6 +59,8 @@ function prettypercent(nominator, denominator)
     else
         str = string(@sprintf("%.2f", value), "%")
     end
-    return lpad(str, 5, " ")
+    # Workaround...
+    str == "100.0%" && (str = "100%")
```

Comment:
Actually, this is why:

```julia
julia> string(@sprintf("%.2f", 9.999), "%")
"10.00%"
```

and above we check the non-truncated value, so something like 9.999 slips through